splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/TargetUrlValidator.java

```diff
@@ -0,0 +1,21 @@
+package com.redhat.console.notifications.splunkintegration;
+
+import org.apache.camel.Exchange;
+import org.apache.camel.Message;
+import org.apache.camel.Processor;
+import org.apache.commons.validator.routines.UrlValidator;
+
+public class TargetUrlValidator implements Processor {
+
+    public void process(Exchange exchange) throws Exception {
+        String[] schemes = { "http", "https" };
+        UrlValidator urlValidator = new UrlValidator(schemes, UrlValidator.ALLOW_LOCAL_URLS);
+
+        Message in = exchange.getIn();
+        String url = in.getHeader("targetUrl", String.class);
```

Comment:
Since this validates `targetUrl`, could we refactor the code after the validation to use `targetUrl` exclusivelly instead of `$simple{headers.metadata[url]}`?

Reply:
Addressed in #66 as it is out of the scope if this PR